deployment/scripts/codeship_heroku_deploy

```diff
@@ -77,6 +82,8 @@ while getopts "N:K:f:u:d:ch" opt; do
       CODESHIP_HEROKU_URL="$OPTARG"; ;;
     c)
       CODESHIP_HEROKU_CHECK_APP_URL=true; ;;
+    C)
+      CODESHIP_HEROKU_CLEAN_DEPLOY=true ;;
```

Comment:
I think we want to default to false for now. To keep current behaviour consistent with what people have right now. Then potentially the env var will be set to true for all new customers once that is added on the Mothership side.

Reply:
We do that right here: https://github.com/codeship-library/heroku-deployment/pull/39/files#diff-4e7cba162f38ceebc26980029f685289R13
Once the -C flag is passed the variable is set to true.

Reply:
My mistake 🙇